solutions/standard/variables.tf

```diff
@@ -38,12 +38,12 @@ variable "prefix" {
 variable "key_protect_instance_name" {
   type        = string
   default     = "base-security-services-kms"
-  description = "The name to give the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing KMS instance. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'."
+  description = "The name to give the Key Protect instance that will be provisioned by this solution. Only used if not supplying an existing Key Protect instance. If a prefix input variable is specified, it's added to the value in the `<prefix>-value` format."
```

Comment:
`Only used if not supplying an existing Key Protect or Hyper Protect Crypto Services instance.`